From 85934a08e7ef2db217c76dbc5cb16e55a81e58ea Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 04 Jun 2019 13:03:40 +0300
Subject: [PATCH] [SW-4185] gaudi: pll: enable w/a for south hbm pll

The South HBM PLL can be switched using the regular W/A for the clock
switch glitch, so enable it.

Change-Id: I85d564b7aaf0464ed089a4720e6e4b86326ffcde
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/gaudi/asic.c b/gaudi/asic.c
index f0d025d..b04622b 100644
--- a/gaudi/asic.c
+++ b/gaudi/asic.c
@@ -120,7 +120,7 @@
 		.name = "HBM", .base = mmPSOC_HBM_PLL_BASE,
 		.nr = 0, .nf = 37, .od = 1, .nb = 18, /* 950 MHz */
 		.div_factor = { 0, 1, -1, -1 },
-		.div_sel = { HL_PLL_PLL_CLK,
+		.div_sel = { HL_PLL_WA_INIT,
 			     HL_PLL_REF_CLK,
 			     HL_PLL_REF_CLK,
 			     HL_PLL_REF_CLK
@@ -250,7 +250,6 @@
 		switch (gaudi_pll_cfg[i].base) {
 		case mmPSOC_CPU_PLL_BASE:
 		case mmPSOC_PCI_PLL_BASE:
-		case mmPSOC_HBM_PLL_BASE:
 		case mmNIC2_HBM_PLL_BASE:
 			continue;
 		default:
